splunk/scripts/pre-run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+export SPLUNK_PASSWORD="Password"
+
+find /opt/splunk/ -type f > /opt/splunk/before.txt
+
+# Running an instance of Splunk during the build preparing the indexed logs for the tests
+/sbin/entrypoint.sh start-service &
+sleep 60
+
+# Verifying the indexing of the logs
+bash /opt/splunk/index_verification.sh
+
+# Option 1: removing only the auth directory with the cacert.pem file
+rm -r /opt/splunk/etc/auth
+
+# Option 2: removing all file additions
```

Comment:
no need for this anymore because of multistage

Reply:
done